### ISSUE TITLE
Analyzing data from multiple files - no example using '?'

### ISSUE THREAD

**Issue by chmille3** [swcarpentry/python-novice-inflammation#382]
I noticed in the Analyzing Data from Multiple Files section, the '*' and '?' were presented for use in pattern strings. However, an example of how the '?' can be used is not given. Basically, it is mentioned but is not used. Two options could be:

- Remove mention of '?', or

- Add a few lines with a simple example: What if one wants the files numbered 10 and up or below 10:

             above 10 - glog.glob(‘data/inflamation-1?.csv’)
             below 10 - glog.glob(‘data/inflamation-0?.csv’)

I hope this is helpful!

**Comment by bittremieux** [comment]
Indeed, using `?` is also mentioned in the "Key Points", but considering that no example or exercise uses it, I think this is a bit confusing. [Glob also supports other wildcards](https://en.wikipedia.org/wiki/Glob_(programming)#Syntax), such as `[...]`, which are not mentioned at all. This seems inconsistent.

`?` is a very useful wildcard, so I'd vote for adding a small example that demonstrates its usage.
